test/WebJobs.Extensions.Tests/PublicSurfaceTests.cs

```diff
@@ -142,9 +140,16 @@ public void HttpPublicSurface_LimitedToSpecificTypes()
                 "HttpExtensionConstants",
                 "HttpJobHostConfigurationExtensions",
                 "AuthorizationLevel",
-                "HttpRouteFactory",
                 "HttpRequestManager",
-                "HttpTriggerAttribute"
+                "HttpTriggerAttribute",
+                "DefaultWebJobsRouteHandler",
```

Comment:
Does this default handler need to be public? When would someone need to reference it?